software/README.md

```diff
@@ -28,3 +28,37 @@ $ pip-compile software/requirements_dev.in
 ```
 
 Both the ``software/requirements_dev.in`` and the generated ``software/requirements_dev.txt`` files should be committed.
+
+## Configuration Customization
+
+Certain properties of the module, such as screen rotation, CPU clock speed, and Raspberry Pi Pico model, can be
+set using a static configuration file.  This file is located at `/config/config_EuroPiConfig.json` on the
+Raspberry Pi Pico. If this file does not exist, default settings will be loaded.  The following shows the
+default configuration:
+```json
+{
+    "europi_model": "europi",
```

Comment:
Is there any way to use Read the Docs style markup for the actual JSON stored on the repo, and then link to that markup in the README? That way the data is always identical but it's much nicer to read.

I'm a total noob when it comes to that stuff, I have no idea how the current docs are compiled but if it's not too hard then that might be a nice middle ground between consistency and readability

Reply:
The generated docs are mostly API related things: https://allen-synthesis.github.io/EuroPi/api.html So I'm not sure that this would fit well there.

That said the github rendering of this markdown file seems fine to me:
https://github.com/Allen-Synthesis/EuroPi/blob/f2ba01942aaf012b249aff9d0385b0e6c0126456/software/README.md

Reply:
I've just found that you can display code snippets in markdown, for example in this branch i just made: https://github.com/roryjamesallen/EuroPi/blob/test-code-snippet/test-code-snippet.md

This example stores the default JSON file as a file on the repo, but with a little rearranging of europi_config.py I think that this could be taken directly from there, meaning no duplicated data and the only thing you need to do is replace the permalink whenever changes to default config settings are made

Reply:
@roryjamesallen The problem is that there is no json in `europi_config.py`. It's python code that defines the config point and its various options. I do like the way that you are thinking though. There is a script that generates a default config for editing and uploading to the hardware. It is possible that we could generate that file during the build process. But that's out of scope for this PR.